test/users.js

```diff
@@ -0,0 +1,159 @@
+        const newUserName = "newUserName";
+        const newEmail = "newEmail@gmail.com";
+
+        const testUserBefore = await models.user.findOne();
+        testUserBefore.userName = newUserName;
+        testUserBefore.email = newEmail;
+
+        const editTestUser = {user: testUserBefore.toJSON()};
+
+        const res = await supertest(app)
+            .put(`/users/${testUserBefore.id}`)            
+            .send(editTestUser)
+            .expect(200);
+
+        const testUserAfter = await models.user.findByPk(testUserBefore.id);
+        
+        assert.equal(testUserAfter.userName, newUserName, "Should update userName");
+        assert.equal(testUserAfter.email, newEmail, "Should update email");
+    });
+});
+
+describe("DELETE /users/", () => {
+    it("should create user, delete him and have status code 200", async () => {
+        const newTestUser = {user: {
+            userName: "91_sava_91",
+            email: "e.gorohovich@eximgarant.by",
+            telephone: "+375 33 641-44-49",
+            password: "55555",
+            firstName: "Evgeny",
+            lastName: "Gorohovich"
+        }};
+
+        const numberOfUsersBefore = await models.user.count();
+        const newUser = await models.user.create(newTestUser.user);
+        
+        const res = await supertest(app)
+            .delete(`/users/${newUser.id}`)
+            .expect(200);
+
+        const numberOfUsersAfter = await models.user.count();
+        const userById = await models.user.findByPk(newUser.id);
+
+        assert.equal(numberOfUsersBefore, numberOfUsersAfter, "Should delete user");
+        assert.equal(userById, null, "Should delete correct user");
+    });
+});
+
+//routes validation tests
+describe("POST /users", () => {
+    it("should return validation error and have status code 400", async () => {
+        const newTestUser = {user: {
+            userName: "su",
+            email: "v.lykov@gmail.com",
+            telephone: "+375 44 757-53-96",
+            password: "44444",
+            firstName: "Slava",
+            lastName: "Lykov"
+        }};
+
+        const res = await supertest(app)
+            .post("/users")
+            .send(newTestUser)
+            .expect(400);
+    });
+});
+
+describe("PUT /users/:id", () => {
+    it("should return validation error and have status code 400", async () => {
+        const newUserName = "newUserName";
+        const newEmail = "newEmail@gmail.com";
+
+        const testUserBefore = await models.user.findOne();
+        testUserBefore.userName = newUserName;
+        testUserBefore.email = newEmail;
+
+        const editTestUser = {user: testUserBefore.toJSON()};
+
+        const res = await supertest(app)
+            .put(`/users/15.5`)            
+            .send(editTestUser)
+            .expect(400);
+    });
+});
+
+describe("DELETE /users/", () => {
+    it.only("should return validation error and have status code 400", async () => {
```

Comment:
Please don't forget to remove .only commands. It's only for debug/local testing purposes. Put this it() under the existing describe